Flick/Cells/ListSummaryCollectionViewCell.swift

```diff
@@ -0,0 +1,274 @@
+        contentView.addSubview(collaborateView)
+        collaborateView.addSubview(collaborateLabel)
+
+        collaboratorsPreviewView = UsersPreviewView(users: collaborators, usersLayoutMode: .collaborators)
+        collaborateView.addSubview(collaboratorsPreviewView)
+
+        privacyLabel.text = isPrivate ? "Only I can view" : "Anyone can view"
+        privacyLabel.textColor = .mediumGray
+        privacyLabel.font = .systemFont(ofSize: 14)
+        contentView.addSubview(privacyView)
+        privacyView.addSubview(privacyLabel)
+
+        lockView.image = UIImage(named: isPrivate ? "lock" : "unlock")
+        privacyView.addSubview(lockView)
+
+        listNameLabel.text = "Foreign Films" // Temp
+        listNameLabel.textAlignment = .center
+        listNameLabel.font = .boldSystemFont(ofSize: 20)
+        contentView.addSubview(listNameLabel)
+
+        let tagCollectionViewLayout = TagFlowLayout(
+            minimumInteritemSpacing: tagCellSpacing,
+            minimumLineSpacing: tagCellSpacing,
+            sectionInset: UIEdgeInsets(top: 2, left: 30, bottom: 5, right: 30)
+        )
+        tagCollectionViewLayout.scrollDirection = .vertical
+        tagCollectionViewLayout.sectionHeadersPinToVisibleBounds = true
+
+        tagCollectionView = UICollectionView(frame: .zero, collectionViewLayout: tagCollectionViewLayout)
+        tagCollectionView.backgroundColor = .offWhite
+        tagCollectionView.register(TagCollectionViewCell.self, forCellWithReuseIdentifier: tagCellReuseIdentifier)
+        tagCollectionView.dataSource = self
+        tagCollectionView.delegate = self
+        tagCollectionView.showsVerticalScrollIndicator = false
+        tagCollectionView.showsHorizontalScrollIndicator = false
+        tagCollectionView.bounces = false
+        tagCollectionView.contentInsetAdjustmentBehavior = .always
+        tagCollectionView.allowsSelection = true
+        contentView.addSubview(tagCollectionView)
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+        let numCollaborators = min(collaborators.count, 8)
+        let fullCollaboratorsWidth = numCollaborators * 20
+        let overlapCollaboratorsWidth = (numCollaborators - 1) * collaboratorsCellSpacing * -1
+        let collaboratorsPreviewWidth = fullCollaboratorsWidth - overlapCollaboratorsWidth
+
+        collaborateLabel.snp.makeConstraints { make in
+            make.centerY.trailing.equalToSuperview()
+        }
+        
+        collaboratorsPreviewView.snp.makeConstraints { make in
+            make.centerY.leading.equalToSuperview()
+            make.trailing.equalTo(collaborateLabel.snp.leading).offset(-8)
+            make.height.equalTo(20)
+            make.width.equalTo(collaboratorsPreviewWidth)
+        }
+
+        collaborateView.snp.makeConstraints { make in
+            make.top.equalTo(listNameLabel.snp.bottom).offset(20)
+            make.centerX.equalToSuperview()
+            make.height.equalTo(20)
```

Comment:
We can probably make the ```20``` a variable right? like all the meta information heights are 20 it seems